Remove test-only makeGenesisBlock utility

Geod24 opened the pull request:
As noted in the comment, now that issue 907 has been fixed,
this can be removed.

Geod24 committed:
Remove test-only makeGenesisBlock utility

As noted in the comment, now that issue 907 has been fixed,
this can be removed.

linked0 reviewed (commented):
Nice, but the `remakeEnrollment` isn't needed anymore also. It can be deleted.

Geod24 committed:
Remove unused & test only remakeEnrollment function

Geod24 commented:
Right, but it seems it was unused before this PR, since I could not find an instance of it in the diff ?
Anyways, added a commit to remove it (and rebased on `v0.x.x`).

linked0 reviewed (approved):
LGTM. Thanks.
